flog/configs/flog.conf

```diff
@@ -0,0 +1,23 @@
+[PATH]
+DATABASE = flog/models/db/posts.db
```

Comment:
in official documentation about Configparser .conf file contains all parameters in lowercase, And .py file in opposite contains all param uppercase(because it  is more readable when you import them in your app, actually they are constants).